test/create-test-transactions.sh

```diff
@@ -3,21 +3,56 @@
 lisk config:set api.nodes http://localhost:4000
 
 NETWORKIDENTIFIER="93d00fe5be70d90e7ae247936a2e7d83b50809c79b73fa14285f02c842348b3e"
-PASSPHRASE="peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready"
+GENESIS_NONCE=103
+MULTISIG_CANDIATE_NONCE=0
+# genesis
+PASSPHRASE1="peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready"
+OPTIONALKEY1="0fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a"
```

Comment:
If this is the optional key from genesis I suggest to rename it for `OPTIONALKEY_GENESIS`